nautobot/core/ui/object_detail.py

```diff
@@ -502,7 +594,10 @@ def render_value(self, key, value, context) -> str:
         - Etc.
         """
         display = value
-
+        try:
+            field_instance = context["obj"]._meta.get_field(key)
+        except FieldDoesNotExist:
+            field_instance = None
```

Comment:
I was trying to keep `KeyValueTablePanel` agnostic of model objects - if so it might be cleaner to implement this case in `ObjectFieldsPanel.render_value` instead, similar to what it's already doing for `render_key`?

Reply:
Ive moved this to part of the acceptance criteria for `UI Components: Key/value model fields`, hence illd resolve it in that PR and remove this changes from this PR

> Key value pairs of model fields - some built in smarts for commonly implemented features
> - ...
> - URL fields should be automatically hyperlinked